content/mqtt/index.textile

```diff
@@ -102,7 +102,7 @@ h2(#use-cases). Recommended usage
 
 We recommend you use MQTT to interact with Ably when:
 
-* Using devices which do not have a native Ably client library, such as Arduino platforms, C/C++ applications, and so on
+* Using devices which do not have an available Ably client library, such as Arduino platforms, C/C++ applications, and so on
```

Comment:
Maybe "device-specific" rather than "available"? 

"We recommend you use MQTT to interact with Ably when:
- The target platform does not have a device-specific Ably client library, such as Arduino, C/C++ applications, and so on
- Bandwidth is limited and you want to keep network traffic to a minimum"

Bonus: parallel bullets 😄

Reply:
Also, Ably Client Library SDK. Once mentioned, can refer to as SDK.

Reply:
Updated the wording around the SDK sentence. I'll leave the rewording of the paragraph to the section review.